src/ScopedCommand.js

```diff
@@ -0,0 +1,26 @@
+import Command from "./Command";
+import minimatch from "minimatch";
+
+export default class ScopedCommand extends Command {
```

Comment:
I don't know if inheritance is the best way to solve this. It should be just as easy to have a util that does this and it will get us away from using inheritance which complicates things